File: task03/go/subtask2.go
package main

import (
    "io"
    "os"
)

func main() {
    input, err := os.Open("input.txt")
    if err != nil {
        panic(err)
    }
    defer input.Close()

    output, err := os.Create("output.txt")
    if err != nil {
        panic(err)
    }
    defer output.Close()

    _, err = io.Copy(output, input)
    if err != nil {
        panic(err)
    }
}

File: task03/rust/subtask2.rs
use std::fs::File;
use std::io::{Read, Write};

fn main() {
    let mut input_file = File::open("input.txt").expect("Error opening input file");
    let mut output_file = File::create("output.txt").expect("Error opening output file");

    let mut string = String::new();
    input_file.read_to_string(&mut string).expect("Error reading input file");
    output_file.write_all(string.as_bytes()).expect("Error writing to output file");
}